cfme/tests/services/test_config_provider_servicecatalogs.py

```diff
@@ -66,6 +68,8 @@ def catalog_item(appliance, request, config_manager, ansible_tower_dialog, catal
     return catalog_item
 
 
+@pytest.mark.meta(automates=[BZ(1717500)])
```

Comment:
@nachandr can you also add this BZ to the `Bugzilla:` docblock?

Reply:
Done